### DISCUSSION TITLE
The integration_test worked, but the android emulator doesn't seem to be running.

### DISCUSSION THREAD

**Discussion by mafreud** [leancodepl/patrol#1631] [Q&A] [answered]
Any suggestions?

### Steps to reproduce
1. clone https://github.com/mafreud/patrol_sandbox
2. launch android emulator (API 33)
3. run ```patrol test --target integration_test/example_test.dart```
4. integration_test passed but emulator not work. 

### Full Code
- https://github.com/mafreud/patrol_sandbox

### example_test.dart
```dart
import 'package:patrol/patrol.dart';
import 'package:patrol_sandbox/main.dart';

void main() {
  patrolTest('demo', nativeAutomation: true, (PatrolTester $) async {
    await $.pumpWidgetAndSettle(const MyApp());
    // prepare network conditions
    await $.native.enableCellular();
    await $.native.disableWifi();

    // toggle system theme
    await $.native.enableDarkMode();

    // handle native location permission request dialog
    await $.native.selectFineLocation();
    await $.native.grantPermissionWhenInUse();

    // tap on the first notification
    await $.native.openNotifications();
    await $.native.tapOnNotificationByIndex(0);
  });
}

```

### patrol version
```yaml
patrol: ^2.2.1
```

### patrol CLI version
```
❯ patrol --version
You're using Patrol CLI 2.0, which has breaking changes.
Read the migration guide at https://patrol.leancode.co/v2.
Disable this warning by setting the PATROL_MIGRATED environment variable.

patrol_cli v2.1.1
```

![2023-08-11 4 26 33 PM](https://github.com/leancodepl/patrol/assets/28733986/56a054f6-ae14-44fc-9d0f-836289526261)

**Comment by bartekpacia** [comment, marked as answer]
The `MainActivityTest.java` was in a wrong directory. See the fix -> https://github.com/mafreud/patrol_sandbox/pull/8

**Reply by mafreud** [reply]
It worked perfectly. Thank you so much!

**Comment by adrian-moisa** [comment]
Believe it or not a single whitespace at the end of the java file extension was the reason I had the same issue. I'm surprised that is even allowed on a mac. May I suggest that you add a check in the code to verify if the `MainActivityTest.java` is found where expected?
